x-pack/plugins/security_solution/public/common/components/sessions_viewer/index.tsx

```diff
@@ -26,8 +26,22 @@ export const defaultSessionsFilter: Required<Pick<Filter, 'meta' | 'query'>> = {
     bool: {
       filter: [
         {
-          exists: {
-            field: 'process.entry_leader.entity_id', // to exclude any records which have no entry_leader.entity_id
+          bool: {
+            // show sessions table results by filtering events where event.action is fork, exec, or end
+            should: [
+              { match_phrase: { 'event.action': 'exec' } },
```

Comment:
Yes that makes sense. I found numerous constants for `process.entry_leader.entity_id` in session view and kubernetes plugins. Would it be better to store the entry leader constant in a  commons constant file `/x-pack/plugins/security_solution/common/constants.ts` that can be used by common Security solution session_viewer component or AWP related plugins? Currently, entry leader constant in session_view or Kubernetes is unreachable.

Reply:
yea, if it's easy to just copy it over to security_solution, that is fine too.

Reply:
i don't know that session_view plugin can reference anything in security_solution as it might create circular dependencies, as security_solution pulls in session_view as a dependency.

Reply:
ideally we could expose that constant in session_view to security_solution. Not sure the exact steps to do that though, maybe @opauloh knows.

Reply:
Ahhh okay I'll sync with him. I also asked a pinged another security solution side. To see what's the correct approach regarding importing constants.

Reply:
Instead of adding 

```js
export const ENTRY_SESSION_ENTITY_ID_FIELD = 'process.entry_leader.entity_id' as const;
```
In the `security_solution` plugin, you can consume exported variables from the  `session_view` plugin common folder anywhere by using:

```js
import { ENTRY_SESSION_ENTITY_ID_PROPERTY } from '@kbn/session-view-plugin/common/constants';
```

About the circular dependency, since `security_solution` is importing methods from `session_viewer` already, it's no problem to import other variables.

![image](https://user-images.githubusercontent.com/19270322/195197730-31ada12d-dead-45a5-8605-0be2e28f1957.png)

A circular dependency problem would only happen in case `session_viewer` imports methods from `security_solution` as well.

![image](https://user-images.githubusercontent.com/19270322/195198267-57f0ae10-de5d-4f3d-a54d-04a196a83d3f.png)

Reply:
Hmm how is the session-view plugin consumed within Security Solution? Session-View plugin import references are not able to consumed the security solution plugin.